website/source/docs/guides/forwarding.html.markdown

```diff
@@ -60,6 +71,15 @@ zone "consul" IN {
 Here we assume Consul is running with default settings and is serving
 DNS on port 8600.
 
+### Dnsmasq
+
+Add the following to your config.  Typically `/etc/dnsmasq.d/` is enabled which should allow creation of a file `/etc/dnsmasq.d/10-consul`:
```

Comment:
I think this could be more clear.  How about something like this:

Dnsmasq is typically configured via files in the `/etc/dnsmasq.d` directory.  To configure Consul, create the file `/etc/dnsmasq.d/10-consul` with the following contents: 

```text
server=/consul/127.0.0.1#8600
```

Once that configuration is created, restart the dnsmasq service.